antora/ec-policies-antora-extension/index.js

```diff
@@ -155,27 +177,12 @@ const helpers = {
     const sortedOutput = output.sort(helpers.annotationSort)
 
     // Group the rules by their package
-    return helpers.groupBy(sortedOutput, a => a.packagePath)
+    const rules = helpers.groupBy(sortedOutput, a => a.packagePath)
 
-  },
-
-  processCollectionLists: (annotationsData) => {
-    const collectionLists = {}
-
-    Object.keys(annotationsData).forEach(packagePath => {
-      annotationsData[packagePath].forEach(a => {
-        (a.collections || []).forEach(c => {
-          (collectionLists[c] ||= []).push({
-            anchor: a.anchor,
-            title: a.title,
-            pkgTitle: a.packageInfo.title,
-            pkgNamespace: a.packageInfo.shortNamespace,
-          })
-        })
-      })
-    })
-
-    return collectionLists
+    return [
+      rules,
+      ruleCollections
```

Comment:
Could we sort by the title? I can see 'slsa2', 'slsa3', 'slsa1' at https://docs-7bn.pages.dev/ec-policies/release_policy#_available_collections .

Reply:
Should be sorted now...